nx/exercises/exercises-1-20.livemd

```diff
@@ -0,0 +1,496 @@
+# Exercises: 1-20
+
+```elixir
+Mix.install([{:nx, "~> 0.6"}])
+```
+
+## Introduction
+
+Inspired by the Python notebook [100 Numpy Exercises](https://www.kaggle.com/code/utsav15/100-numpy-exercises/notebook).
+
+## 1-10
+
+**1. Install `Nx` in a Livebook. (★☆☆)**
+
+```elixir
+# Add your solution here.
+```
+
+<details class="rounded-lg border-2 border-green-bright-300 p-2">
+  <summary class="cursor-pointer font-bold mx-2">Example solution</summary>
+  <div class="p-2">
+
+  ```elixir
+  Mix.install([{:nx, "~> 0.6"}])
+  ```
+
+  </div>
+</details>
+
+<!-- livebook:{"break_markdown":true} -->
+
+**2. Create a 1-D tensor of 10 zeros. (★☆☆)**
+
+```elixir
+# Add your solution here.
+```
+
+<details class="rounded-lg border-2 border-green-bright-300 p-2">
```

Comment:
Ah btw, ideally we should not rely on tailwind classes. Tailwind prunes classes at build time, so whether some of them are available depends on whether they are used anywhere in Livebook. This is not critical, but even more relevant concern is that we want to render this using ExDoc and it's not going to translate.

We don't have to be fancy with it, personally I actually prefer this simpler version:

````
<details>
  <summary style="cursor: pointer; font-weight: bold;">Example solution</summary>
  <div style="padding-top: 8px">

  ```elixir
  ...
  ```

  </div>
</details>
````

It doesn't bring too much attention. Currently the green border makes it the first thing my eyes focus on, and we shouldn't disrupt the reading flow :)

Reply:
> This is not critical, but even more relevant concern is that we want to render this using ExDoc and it's not going to translate.

Oh good point. I was actually choosing Tailwind classes based on what happened to be available, which felt brittle.

> Currently the green border makes it the first thing my eyes focus on, and we shouldn't disrupt the reading flow :)

Ha that's understandable. I even tried to tone it _down_ from the original:

https://the-eef.slack.com/archives/C01PU0RDJ8L/p1699750471235479?thread_ts=1699720134.906519&cid=C01PU0RDJ8L